hack/make-rules/verify.sh

```diff
@@ -69,11 +78,17 @@ function is-quick {
 }
 
 function run-cmd {
+  set +o nounset
```

Comment:
This is smelly

Reply:
It is, but seems to be a requirement for the forked shell2junit code. We already do it here: https://github.com/kubernetes/kubernetes/blob/884e4b7b9c63a9a3b94278972d640454b8d16e47/hack/make-rules/test-cmd-util.sh#L91-L114

Reply:
Maybe we shouldn't fork a lib that isn't hygenic? :)